client/src/ui-components/CreateProductForm/CreateProductForm.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Form, Container } from "react-bootstrap";
+import "./CreateProductForm.css";
+import Product from "../../services/models/Product";
+import Button from "../Button/Button";
+function CreateProductForm(props) {
+  const handleChange = (e) => {
+    props.setCreatedProduct({
+      ...props.createdProduct,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  return (
+    <Container>
+      <h1 className="title">Create Product</h1>
+      <Container className="createProductForm">
+        <Form onSubmit={props.handleSubmit}>
+          <Form.Group
+            className="mb-3"
+            value={props.createdProduct.name}
+            onChange={handleChange}
+          >
+            <Form.Label>Name</Form.Label>
+            <Form.Control
+              required
+              name="name"
+              type="text"
+              placeholder="Enter name of the product"
+            />
+          </Form.Group>
+
+          <Form.Group
+            className="mb-3"
+            value={props.createdProduct.description}
+            onChange={handleChange}
+          >
+            <Form.Label>Description</Form.Label>
+            <Form.Control
+              required
+              name="description"
+              as="textarea"
+              placeholder="Enter some description for product"
+              rows={3}
+            />
```

Comment:
as above, add `maxLength="100"`
because in productDescriptionBodyValidator we set that .isLength({ max: 100 })